Specification.md

```diff
@@ -0,0 +1,17 @@
+### plugins.yaml
+
+The `plugins.yaml` file contains a list of objects, each representing a plug-in.
+
+YAML has been chosen because of its good readability while still being machine-readable and reasonably widespread. Compatibility with other formats is achieved through the CI, which converts the YAML file to other formats.
+
+### Plug-Ins
+
+A Plug-In object consists of the following key-value pairs:
+
+- `name`
+- `url`: A link to the project's github repository.
+- `version`: Preferably the name of a git tag, or a commit hash.
+- `author`: The author(s) of the plug-in.
```

Comment:
Is this a single value or is a list allowed?

Reply:
It's a string, how you format that string is up to you. `Amazinite & Contributors` would be perfectly acceptable.